index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
+    <title>Survey form</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<main>
+<body>
+<h1 id="title">Survey Form</h1>
+<p id="description">Please complete this survey</p>
+<form id="survey-form">

```

Comment:
Please use class instead of ID its better to work

Reply:
- [ ] Please kindly explain in more detail why you suggest that change, always includes an explanation, a "why", for each requested change 😁 🙃. Also, use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.